README.md

```diff
@@ -1,56 +1,62 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Space Traveller's Hub
 
-# Project Name
-
-> Description the project.
+> A simple and interractive React App that utilises an external `space X` API to provide information about space rockets and missions. User can reserve rockets, join mission and these can be viewed in a the user's profile. A user can also read more about the rockets and missions.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+- HTML, CSS, ReactJs
+- React Router, Redux Thunk, Jest
+- NodeJS
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
+- You need to have a nodejs installed on your machine.
+- You need to have a code editor installed on your local machine. I recommend using [VSCode](https://code.visualstudio.com/).
 
 ### Setup
+- Clone the repository `git@github.com:rotshidzw/space-travelers-hub.git` to your local machine.
+- Open the terminal and navigate to the root of the repository.
+- Run the following command:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
 
-### Install
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
 
-### Usage
+### `npm test`
 
-### Run tests
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
 
-### Deployment
+### `npm run build`
 
+Builds the app for production to the `build` folder.\
+It correctly bundles React in production mode and optimizes the build for the best performance.
+
+The build is minified and the filenames include the hashes.\
+Your app is ready to be deployed!
+
+See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
 
 
 ## Authors
 
-👤 **Author1**
+👤 **Author 1**
```

Comment:
- [ ] OPTIONAL
Kindly replace the text **Author1** **Author2** on _line 51_ and _line 56_ of your ReadMe with your your full names, so as to make your ReadMe more professional